app/src/main/java/com/mstudio/superheroarch/MainActivity.kt

```diff
@@ -16,5 +19,18 @@ class MainActivity : AppCompatActivity() {
             v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
             insets
         }
+        
+        val mainButton : Button = findViewById(R.id.main_button)
+        val mainText = findViewById<TextView>(R.id.title)
+
+        mainButton.setOnClickListener{
+            val handler = Handler()
```

Comment:
you don't need the handler or runnable to comply with the requirement of changing the text when tapping the buttob